scripts/build.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$root = realpath(__DIR__ . '/../') . '/';
+
+$archiveName = 'phpmd.phar';
+$version = parse_ini_file($root . 'build.properties')['project.version'] ?? '@package_version@';
```

Comment:
```suggestion
$changelog = file_get_contents(__DIR__ . '/CHANGELOG');
$version = preg_match('/phpmd-([\S]+)/', $changelog, $match) ? $match[1] : '@package_version@';
```

Would allow to completely remove `build.properties`

Reply:
I will merge as is and then bring this up as a proposal

Reply:
https://github.com/phpmd/phpmd/pull/1206